demo/react/hooks/demo/index.js

```diff
@@ -19,6 +19,7 @@ import UseScrollDemo from './UseScrollDemo'
 import UseStepsDemo from './UseStepsDemo'
 import UseSwipeDemo from './UseSwipeDemo'
 import UseToggleDemo from './UseToggleDemo'
+import UseControlledState from './UseControlledState'
```

Comment:
Just a small point, would be nice to keep the files naming consistent with the other hook demos :)

Reply:
done!